addons/account/models/decimal_precision.py

```diff
@@ -0,0 +1,9 @@
+from odoo import models
+
+class DecimalPrecision(models.Model):
```

Comment:
Hi. Why put this function in account ? 
It can be usefull out of this context. Dont you think ?

except that point thanks for this patch !

Reply:
The model and the function are in `base`, but the "Discount" string is in `account`

Reply:
Not sure to understand the _"Discount" string_ You talk.

Some 'discount' fields, are present in module that doesn't depends of account, like product : 

https://github.com/odoo/odoo/blob/17.0/addons/product/models/product_supplierinfo.py#L64

Reply:
Yes, `product` is the only module not depending on `account` using that decimal precision.
Do you have a use case of a module depending on `product` but not on `account` where the decimal precision is used and should be avoided?

Reply:
Hi. Thanks for your reply. 

> Do you have a use case of a module depending on product but not on account where the decimal precision is used and should be avoided?
No it was a theoritical question about design.
For me, that code should be in base module, to be used everywhere. But indeed, no real use case with real problem.